### ISSUE TITLE
VideoA11y: Method and Dataset for Accessible Video Description

### ISSUE THREAD

**Issue by arataito17** [SeitaKayukawa/Blind_Accessibility_Papers#641]
### Links
- Paper: https://arxiv.org/abs/2502.20480
- Movie:
- Project Page: https://people-robots.github.io/VideoA11y/

### Abstract
Video descriptions are crucial for blind and low vision (BLV) users to access visual content. However, current artificial intelligence models for generating descriptions often fall short due to limitations in the quality of human annotations within training datasets, resulting in descriptions that do not fully meet BLV users’ needs. To address this gap, we introduce VideoA11y, an approach that leverages multimodal large language models (MLLMs) and video accessibility guidelines to generate descriptions tailored for BLV individuals. Using this method, we have curated VideoA11y-40K, the largest and most comprehensive dataset of 40,000 videos described for BLV users. Rigorous experiments across 15 video categories, involving 347 sighted participants, 40 BLV participants, and seven professional describers, showed that VideoA11y descriptions outperform novice human annotations and are comparable to trained human annotations in clarity, accuracy, objectivity, descriptiveness, and user satisfaction. We evaluated models on VideoA11y-40K using both standard and custom metrics, demonstrating that MLLMs fine-tuned on this dataset produce high-quality accessible descriptions. Code and dataset are available at https://people-robots.github.io/VideoA11y/
### Approach
本研究では、視覚障害者向けに映像内容を分かりやすく伝えるため、マルチモーダル大規模言語モデル（MLLM）と音声説明（AD）のプロフェッショナル向けガイドラインを組み合わせた新手法「VideoA11y」を提案している。まず、オンライン上の複数の信頼性ある情報源から抽出した42項目のADガイドラインを用いて、MLLMに適したプロンプトを設計。次に、映像から重要なシーン変化を捉えるキー・フレームを抽出し、必要に応じて人間による注釈も活用。これらの情報を基に、MLLM（例：GPT-4V）を用いて、BLVユーザー向けに最適化された詳細で正確な映像説明を自動生成または既存の説明を改訂する。さらに、この手法を用いて40,000本の映像に対する説明を収録した大規模データセット「VideoA11y-40K」を構築し、様々な評価実験を通じてその有効性を検証している。

 ### Strong Point

- ガイドラインに基づく精度の高い説明生成

プロのADガイドライン42項目を活用することで、映像内容を正確かつ詳細に記述する説明が生成可能。

- 大規模データセットの構築

40,000本の映像を対象とした「VideoA11y-40K」を構築し、幅広いジャンルに対応することで、実用性と汎用性を向上。

- 多角的な評価実験

視覚正常者、視覚障害者、プロの音声説明者の3者による評価を実施し、定量的・定性的に手法の優位性を検証。

- 先進的なMLLMの活用

最新のMLLM（例：GPT-4V）を採用することで、従来の人間注釈や初心者による説明に比べ、高品質な説明生成が実現。
### Evaluation
本研究では、以下の5つのユーザースタディおよび技術実験を実施して評価を行った：

- Study 1

150名のMTurk参加者を対象に、オープンソースおよびプロプライエタリMLLMによる生成説明を、記述性、客観性、正確性、明瞭性の4評価指標で比較評価。

- Study 2

初心者の人間注釈と比較し、VideoA11yによる説明の質の向上を明らかに。

- Study 3

熟練した説明者が作成した高品質な説明と、VideoA11y生成説明との比較評価を実施（対象動画47本）。

- Study 4

プロの音声説明者7名による評価で、両者の説明の質を詳細に検証。

- Study 5

40名の視覚障害者を対象に、VideoA11y説明と従来の人間注釈を比較し、理解度や満足度を評価。
さらに、標準的なNLPメトリクスおよびカスタムメトリクスを用いて、ファインチューニング前後のモデル性能を定量的に評価した。

### Result

- VideoA11yによる映像説明は、初心者の人間注釈を大幅に上回り、熟練した説明者が作成した高品質な説明と同等以上の性能を示した。

- 40,000本の映像を収録したVideoA11y-40Kデータセットを用いてファインチューニングしたモデルは、記述性、客観性、正確性、明瞭性の各評価指標において、ベースラインモデルと比較して有意な改善が確認された。

- 技術実験では、特に正確性と明瞭性において、VideoA11yを導入することで大幅な性能向上が得られることが示された。

### Limitation & Future Work
Limitation

- ハルシネーションのリスク

人間注釈が存在しない場合、ADガイドラインのみの依存により、映像に実際には存在しない情報が含まれる（ハルシネーション）の可能性がある。

- 注入攻撃への脆弱性

悪意のある人間注釈が混入することで、誤った情報が拡散されるリスクが存在する。

- パーソナライゼーションの不足

現在の手法は、利用者個々の好みや必要性に応じたカスタマイズが十分に行われておらず、個別対応が課題。

- 映像との連携の最適化不足

Future work

映像の流れに合わせたタイミングや説明の長さの制御が十分ではなく、インライン説明の実現が課題となっている。
今後の研究では、これらの課題に対して、ハルシネーションの低減策、注入攻撃への耐性強化、利用者ごとのパーソナライズ機能の拡充、さらに映像のシーンに適したタイミングでのインライン説明生成など、より実用的で安全な映像説明システムの構築が期待される。

![Image](https://github.com/user-attachments/assets/ee563647-e5cb-40ae-b1aa-7954ed690fc0)
